app/uk/gov/hmrc/ccn2connectivitytester/models/SealedTraitJsonFormatting.scala

```diff
@@ -0,0 +1,33 @@
+/*
```

Comment:
Might be best to grab this from common domain even though I know you're not using any of the other contents.  That way if we fix it later....